src/main/resources/i18n/messages.properties

```diff
@@ -1544,6 +1564,49 @@ pipeline.launch.page-title=Pipelines - Launch
 pipeline.launch.title=Select a Pipeline
 pipeline.launch.auto-description=Select a pipeline for automated analysis.
 
+LaunchButton.submit=Launch Pipeline
+LaunchButton.submit-automated=Create Automated Pipeline
+LaunchContent.noSamples.message=No samples are in the cart.
+LaunchContent.noSamples.description=In order to run a pipeline, go to a project and add the samples you want to run to the cart.
+
+LaunchDetails.label=General Details
+LaunchDetails.name=Name
+LaunchDetails.name.required=Workflow Pipeline require a name for easy identification
+LaunchDetails.description=Description
+LaunchDetails.emailResults=Email Pipeline Results
+
+LaunchParameters.title=Parameters
+SavedParameters.title=Select a pre-defined parameter set
+SavedParameters.tooltip=Default Parameters are pipeline defaults
+SavedParameters.modified=This template has been modified.
+SavedParameters.modified.description=You can save this as a new parameter template so they are quickly available to you for future runs.
+SavedParameters.modified.saveAs=Save Template As
+SavedParameters.modified.name=Template Name
+SavedParameters.modified.save=Save
+SavedParameters.modify=Modify
+SavedParameters.modified-name.required=Please enter a name for these parameters
+SavedParameters.modified-name.length=Name must be at least 3 characters long.
+ParametersModal.modified=MODIFIED
+
+ReferenceFiles.label=Reference Files
+ReferenceFiles.required=A reference file is required to run this pipeline.  Please add one directly to the pipeline below or through on of the projects.
```

Comment:
Looks like a typo in `through on of the projects`. Should be `through one of the projects`

Reply:
Updated in 554cc1efc6017525cafb142e205f6dc6125d0a25